invenio_notifications/config.py

```diff
@@ -1,9 +1,29 @@
 # -*- coding: utf-8 -*-
 #
 # Copyright (C) 2023 CERN.
+# Copyright (C) 2023 Graz University of Technology.
 #
 # Invenio-Notifications is free software; you can redistribute it and/or
 # modify it under the terms of the MIT License; see LICENSE file for more
 # details.
 
 """Invenio module for notifications support."""
+
+from flask_babelex import gettext as _
+
+from .backends.email import EmailNotificationBackend
+
+NOTIFICATIONS_BACKENDS = {
```

Comment:
Would it make sense to have this just int he config below? or does it need to be exposed?

```python
lass NotificationConfig:
     """Config."""

     backends = {
        EmailNotificationBackend.
     }
```

Reply:
It can probably be moved. If only the backends should be overwritten but everything else should stay the same, one can always inherit from the base `NotificationConfig` and only change the backends.